canonical-web-and-design/jaas-dashboard
# Add ContentReveal to Models entity view

## Done

Added `ContentReveal` around tables on model entity page

## QA

- Run feature branch using Juju controller with a model with cross-model relations
- Go to model with CMR
- Verify content reveal blocks appear
- Go to model without CMR
- Verify there is no content reveal blocks
- Click around, ensure nothing breaks

Note: Github has become confused with the git log.. will auto-resolve on merge.

Fixes https://github.com/canonical-web-and-design/juju-squad/issues/1689
